Sources/MapboxMaps/Gestures/GestureHandlers/GestureHandler.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+internal class GestureHandler {
```

Comment:
nit: Move internal modifiers to each item below as opposed to on the class

Reply:
This was a simple move, so this existed earlier.